release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2066,6 +2066,17 @@ In the table below, features are marked with the following statuses:
 [id="ocp-4-6-known-issues"]
 == Known issues
 
+* In {product-title} 4.6.8, a bug fix was introduced that changed how certificates were regenerated by the Cluster Logging Operator (CLO). This fix causes an issue where the CLO could possibly regenerate certificates while the Elasticsearch Operator (EO) is trying to restart the cluster. This results in communication issues between the EO and the cluster, causing EO nodes to have mismatched certificates.
++
+The mismatched certificates could cause issues when upgrading Elasticsearch. As a workaround, you can selectively upgrade the CLO followed by the EO, or vice versa. If that does not work, run the following command to restart Elasticsearch pods:
```

Comment:
typically don't use Latin in technical docs, but I don't see any prohibition on "vice versa" specifically in our style guide (only mentions Latin abbreviations like i.e. and e.g.). Just noting because it seems wrong to me, take that as you like :smile: Could say to do the upgrade one at a time or something to avoid.

Reply:
I had the exact same though, and then saw ISG use it repeatedly throughout their guide :sweat_smile:. I'll just go ahead and avoid it to be safe.